angr/calling_conventions.py

```diff
@@ -253,11 +254,22 @@ def __init__(self, arch, args=None, ret_val=None, sp_delta=None, func_ty=None):
         :param args:        A list of SimFunctionArguments describing where the arguments go
         :param ret_val:     A SimFunctionArgument describing where the return value goes
         :param sp_delta:    The amount the stack pointer changes over the course of this function - CURRENTLY UNUSED
-        :parmm func_ty:     A SimType for the function itself
+        :parmm func_ty:     A SimTypeFunction for the function itself, or a string that can be parsed into a
```

Comment:
Perhaps add here an example of the kind of string to pass in? There are lots of ways to declare function-like things in C, but we want basically a full function prototype declaration.